src/main/cli/index.ts

```diff
@@ -218,7 +218,7 @@ const yargsInit = () =>
                 if (openPublicationRequestedBool) {
 
                     // flush session because user ask to read a publication
-                    flushSession();
+                    await flushSession();
```

Comment:
`flushSession` doesn't need to be `async` i guess

https://github.com/edrlab/thorium-reader/blob/8069ae9d3b6c52036304d79cb35bb0886428031e/src/main/tools/flushSession.ts#L11

Reply:
Making it non-async feels out of scope for this particular PR.

However, it being async doesn't matter – it acts like a synchronous function since there are no continuation points (`await`s) within.